### ISSUE TITLE
"Open clipboard URL" strings without http (https) prefix

### ISSUE THREAD

**Issue by HAMiDY25** [Robbendebiene/Gesturefy#579]
Does not work with capital letters.🤔

**Google.com**
**google.Com**
**Github.com**

Long address does not work too, does the problem relate to this issue?
**`github.com/Robbendebiene/Gesturefy`**

### Your System
- Operating system: Windows 10 LTSC 1809
- Firefox version: 86.0
- Gesturefy version: 3.1.1

**Comment by Robbendebiene** [comment]
Thanks for reporting this.

> Long address does not work too, does the problem relate to this issue?
> github.com/Robbendebiene/Gesturefy

This is not caused by the capital letters, but rather by design. If you allow URLs without protocol it becomes hard to distinguish them from normal text. That's probably why Firefox won't open domains with a path either. However I may tweak this a little bit since I agree that if one can open `github.com` then `github.com/Robbendebiene/Gesturefy` should also be possible.

**Comment by Robbendebiene** [comment]
After investigating this again I remembered why I did follow Firefox's behaviour. It's hard to decide where to draw the line. For example if I allow `github.com/Robbendebiene/Gesturefy`why not `github.com?something=12`. It's basically impossible to avoid false positives but the amount grows if I weaken the checks. So I understand that this might not seem reasonable to everyone but in this case I like to stick to Firefox's behaviour and only allow plain domains like `www.example.com`.

But of course I'll fix the bug regarding case sensitive letters!
